packages/main/src/DateRangePicker.js

```diff
@@ -198,6 +206,11 @@ class DateRangePicker extends DatePicker {
 			timestamp = focusTimestamp || oCalDate.valueOf() / 1000,
 			dates = this._splitValueByDelimiter(this.value);
 
+		if (this._initialRendering) {
+			this._oneTimeStampSelected = dates[0].trim() === dates[1].trim();
```

Comment:
Maybe it is good to declare this flag in the constructor "this._oneTimeStampSelected"  with short comment what is used for